tsconfig.json

```diff
@@ -15,6 +15,6 @@
 		"jsx": "preserve",
 		"baseUrl": "."
 	},
-	"exclude": ["node_modules"],
+	"exclude": ["node_modules", "tests/e2e/helper/*.ts"],
```

Comment:
Reason for exclusion?

Reply:
Not to break the default build . Without this removal the default build will not run.